yoti-sdk-spring-boot-example/src/main/resources/templates/fragments.html

```diff
@@ -2,10 +2,12 @@
 <html xmlns:th="http://www.thymeleaf.org">
 <head/>
 <body>
-    <div class="" th:fragment="anchor">
-
-        Hello!
-
+<td th:fragment="anchorDisplay(anchors)">
+    <div th:each="anchor : ${anchors}" style="border: 1px solid grey; display:inline-block; margin-bottom=0" >
+        <p style="line-height:0"><i>Type:</i> <span th:text="${anchor.type}"/></p>
+        <p style="line-height:0"><i>Value:</i> <span th:text="${anchor.value}"/></p>
+        <p style="line-height:0"><i>SubType:</i> <span th:text="${anchor.subType}"/></p>
+        <p style="line-height:0"><i>Timestamp:</i> <span th:text="${anchor.signedTimestamp.timestamp.date}"/>, <span th:text="${anchor.signedTimestamp.timestamp.time}"/></p>
```

Comment:
I don't believe we have timestamp in the other SDK profile pages. It is a nice addition to have, I'm just thinking that the profile page is quite crowded already.

Reply:
Well, we return it.  So let's go with it for now.  Whether we keep it in the new demo design is another matter.  This is just an interim change.